packages/@o3r/test-helpers/src/utilities/locker.ts

```diff
@@ -100,8 +100,12 @@ export class Locker {
           resolve();
         }
       };
-      watcher = watch(this.options.lockFilePath, {persistent: false}, check);
-      watcher.on('error', check);
+      try {
+        watcher = watch(this.options.lockFilePath, {persistent: false}, check);
+        watcher.on('error', check);
```

Comment:
does it make sense to keep this line ? it is suppose to do what we do in the catch, or I am wrong :D

Reply:
the reason I added a try/catch is for the rare case when the file has already been deleted in parallel when we start the watch (file not found exception)
the `on('error')` catch errors after the watch has been started
I would say we still need both